js/articleView.js

```diff
@@ -70,15 +71,23 @@ articleView.handleCategoryFilter = function() {
   $('#category-filter').on('change', function () {
     var selection = $(this).val();
     if (selection) {
-      $('#blogContainer article').hide();
-      $('#blogContainer article[data-category="' + selection + '"]').fadeIn(800);
+      $('#blog article').hide();
+      $('#blog article[data-category="' + selection + '"]').fadeIn(800);
     } else {
-      $('#blogContainer article').not('.template').show();
+      $('#blog article').not('.template').show();
     }
   })
 };
 
```

Comment:
In the above 2 functions you refer to ".not('.template')" but I don't see that class anywhere; wasn't it removed when you implemented handlebars? Can this code be removed too?